ternal_apriljoy/restapi/ternal_apriljoy_backend/update_opmband.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+function updateOpmBand($servername, $username, $password, $dbname, $data)
+{
+    $connect = mysqli_connect($servername, $username, $password, $dbname);
+    if (!$connect) {
+        die("Connection failed: " . mysqli_connect_error());
+    }
+
+    foreach ($data as $key => $value) {
+        $data[$key] = mysqli_real_escape_string($connect, $value);
+    }
+
+    $sql = "UPDATE opmBand
+        SET opmband_name = '$data[opmband_name]', origin = '$data[origin]', 
+        leadvocalist_name = '$data[leadvocalist_name]', formation_date = '$data[formation_date]', 
```

Comment:
```suggestion
        leadvocalist_name = '$data[leadvocalist_name]', formation_date = '$data[formation_date]', 
```
 leadvocalist_name = '$data[leadvocalist_name]', 
        formation_date = '$data[formation_date]', 
        first_album = '$data[first_album]'